remoteit/desktop
# DESK-1037 DESK-1085 public connections

### Description



- Fix clearing errors to only happen after install
- Fix certificate inclusion for mac
- Adding support for public connections
- Trying to fix missing h1 font loading
- Fixing typescript issues
- Don't show share button on shared devices
- Allow clearing of connections without loosing state
- Show disabled settings as lighter grey
- DESK-1085 LAN sharing not indicating when off
- Fixing reset connection names generation


### Test plan


Testing proxy only connections by enabling test ui then:

1. Click new connection and select a web service
2. Click the shareable link toggle at the bottom
3. Click start connection
4. You should see the public proxy connection details

### Ticket



<https://remoteit.atlassian.net/browse/DESK-1037>
<https://remoteit.atlassian.net/browse/DESK-1085>
